vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -14969,6 +15010,15 @@ unit:OHM-M
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Ohm Meter" ;
 .
+unit:OHM-M2-PER-M
```

Comment:
I suppose this is OK. We already have the OHM-M as a unit of Resistivity.

Reply:
I know ...